go/p2p/host.go

```diff
@@ -197,3 +206,28 @@ func (cfg *ConnGaterConfig) Load() error {
 
 	return nil
 }
+
+// NewConnManager constructs a new connection manager.
+func NewResourceManager() (network.ResourceManager, error) {
+	// Use the default resource manager for non-seed nodes.
+	if config.GlobalConfig.Mode != config.ModeSeed {
+		return nil, nil
+	}
+
+	// Tweak limits for seed nodes.
+	defaultLimits := rcmgr.DefaultLimits
+	defaultLimits.SystemBaseLimit.ConnsInbound = rcmgr.DefaultLimits.SystemBaseLimit.ConnsOutbound
+	defaultLimits.SystemLimitIncrease.ConnsInbound = rcmgr.DefaultLimits.SystemLimitIncrease.ConnsOutbound
```

Comment:
Should `StreamsInbound` also be updated, as it is smaller compared to `StreamsOutbound`?